### DISCUSSION TITLE
Keeping track of ideas for future development

### DISCUSSION THREAD

**Discussion by CraigClark** [bcgov/MFIN-Data-Catalogue#38] [Ideas]
@NicoledeGreef 

For tracking ideas for the project, how about we make another issue template. We create a tag called _idea_ is automatically assigned when the issue is created.

An _idea_ issue would have a section for a user story, As a… I want to… so that…. and an additional info section.

Ideas do not get assigned a project, this will prevent polluting the project board. 

If an idea gets promoted to feature, point we characterize the feature as we would anything else, and close the idea.

What do you think?

**Comment by NicoledeGreef** [comment]
Thanks @CraigClark - let’s try that!

**Reply by CraigClark** [reply]
This is implemented. To use it, create an issue, select idea, fill in the blanks
